disposable_email_blacklist.conf

```diff
@@ -112,11 +122,13 @@
 9ox.net
 9q.ro
 a-bc.net
+a.a.fbmail.usa.cc
```

Comment:
Here just add `fbmail.usa.cc` as `usa.cc` is a free domain registration and we don`t want to blacklist all other domains.